opennlp-uima/src/main/java/opennlp/uima/normalizer/NumberUtil.java

```diff
@@ -52,31 +51,36 @@ public static boolean isLanguageSupported(String languageCode) {
   }
 
   /**
-   * Removes trailing and containing space.
+   * Removes a character from given string {@code s}.
+   *
+   * @param s The string to process.
+   * @param remove The {@link Character} to remove from {@code s}.
    */
-  private static String removeChar(String string, char remove) {
+  private static String removeChar(String s, char remove) {
```

Comment:
This is basically a `s.replace(remove, "")` (replace all occurences of the given `char` in the given `String` literal). Sadly, were is no `String#replace(char,String)` available, but we could do `s.replace(Character.toString(remove), "")` (or change  the method signature).

Reply:
Alternative: As the method is `private` and only used for removing `' '` from a given String, we could avoid dealing with the [hell of whitespace](https://en.wikipedia.org/wiki/Whitespace_character) andjust replace it with a call to `s.replaceAll("\\s+", "");` ?

Reply:
Since that's more than a Javadoc change and so we can merge this, I wrote [OPENNLP-1437](https://issues.apache.org/jira/browse/OPENNLP-1437) to change the function.